src/adapter/src/client.rs

```diff
@@ -396,144 +398,221 @@ impl SessionClient {
     /// objects.
     pub async fn simple_execute(
         &mut self,
-        stmts: &str,
+        requests: Vec<(&str, Vec<&str>)>,
```

Comment:
nit: can we introduce a `SimpleExecuteRequest` type with `statement` and `params` fields?

Reply:
See `HttpSqlRequest` in last commit